### ISSUE TITLE
New release to pick up recently-added widgets 

### ISSUE THREAD

**Issue by mwcraig** [pbugnion/ipywidgets_server#36]
There are some new widgets (e.g. `FloatLogSlider`) that aren't included in the javascript bundled with the last release of `ipywidgets_server`.  Looks like a new release (and maybe bumping the minimum npm version of `@jupyter-widgets/controls`) will do the trick.

This code snippet will demonstrate the problem; when you try connecting to port 8866 you just get a blank page, and errors in the console about not being able to find `FloatLogSliderModel`:

```python
# Add a slider to allow adjusting MS distance brightness

from ipywidgets import FloatLogSlider
initial_dist = 10

dist_slider = FloatLogSlider(base=10, min=1, max=4, step=0.01,
                             description='Distance (pc)', value=3,
                             readout_format='d')

```

Pinging @JuanCab who found the issue.

**Comment by pbugnion** [comment]
Good spot! Thanks for the reproducible example.

The dependencies are hard-coded in the `package.json`.

**Comment by mwcraig** [comment]
> The dependencies are hard-coded in the package.json.

Yeah; I think doing a new ipywidgets_server release would pick up the latest ipywidgets js.
